vignettes/articles/tbl_ard-functions.Rmd

```diff
@@ -0,0 +1,125 @@
+---
+title: "Building Tables using tbl_ard*()"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+For most tables built using {gtsummary}, ARDs for quality control or table summaries can be extracted. However, there are instances where an ARD first approach is beneficial as building more complex tables may require analysis of the data structures prior to table construction. For this ARD first approach, {gtsummary} has `tbl_ard*()` functions that will generate a tables.
```

Comment:
For the prose, can you put each sentence on its own line? This will make git diffs easier to read in the future